src/jmcomic/cli.py

```
@@ -102,6 +102,10 @@ def run(self, option):
         from .api import download_album, download_photo
         from common import MultiTaskLauncher
 
+        if len(self.album_id_list) == 0 and len(self.photo_id_list) == 0:
+            print('未指定任何 id，请提供 album 或 photo 的 id，例如: jmcomic 123')
```

Ruff (0.15.21)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)